src/datashare/azext_datashare/manual/commands.py

```diff
@@ -0,0 +1,137 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+# pylint: disable=line-too-long
+# pylint: disable=too-many-statements
+
+from azure.cli.core.commands import CliCommandType
+
+
+def load_command_table(self, _):
+
+    from azext_datashare.generated._client_factory import cf_account
+    datashare_account = CliCommandType(
+        operations_tmpl='azext_datashare.vendored_sdks.datashare.operations._account_operations#AccountOperations.{}',
+        client_factory=cf_account)
+    with self.command_group('datashare account', datashare_account, client_factory=cf_account) as g:
+        g.custom_command('list', 'datashare_account_list')
+        g.custom_show_command('show', 'datashare_account_show')
+        g.custom_command('create', 'datashare_account_create', supports_no_wait=True)
+        g.custom_command('update', 'datashare_account_update')
+        g.custom_command('delete', 'datashare_account_delete', supports_no_wait=True, confirmation=True)
+        g.wait_command('wait')
+
+    from azext_datashare.generated._client_factory import cf_consumer_invitation
+    datashare_consumer_invitation = CliCommandType(
+        operations_tmpl='azext_datashare.vendored_sdks.datashare.operations._consumer_invitation_operations#ConsumerInvitationOperations.{}',
+        client_factory=cf_consumer_invitation)
+    with self.command_group('datashare consumer invitation', datashare_consumer_invitation, client_factory=cf_consumer_invitation) as g:
+        g.custom_command('list', 'datashare_consumer_invitation_list')
+        g.custom_show_command('show', 'datashare_consumer_invitation_show')
+        g.custom_command('reject-invitation', 'datashare_consumer_invitation_reject_invitation')
+
+    from azext_datashare.generated._client_factory import cf_data_set
+    datashare_data_set = CliCommandType(
+        operations_tmpl='azext_datashare.vendored_sdks.datashare.operations._data_set_operations#DataSetOperations.{}',
+        client_factory=cf_data_set)
+    with self.command_group('datashare dataset', datashare_data_set, client_factory=cf_data_set) as g:  # modified
+        g.custom_command('list', 'datashare_data_set_list')
+        g.custom_show_command('show', 'datashare_data_set_show')
+        g.custom_command('create', 'datashare_data_set_create')
+        g.custom_command('delete', 'datashare_data_set_delete', supports_no_wait=True, confirmation=True)
```

Comment:
Curious about why "create" does not support no wait. Does it indicate "create" is faster than "delete"?

Reply:
I guess delete involves updating for all consumers who received the dataset and may have created dataset-mapping, trigger related to it.